### DISCUSSION TITLE
Ability to add extra white space around a PDF page

### DISCUSSION THREAD

**Discussion by BitzSec** [xournalpp/xournalpp#4245] [Q&A]
Hello everyone,

First of all, I would like to express my huge appreciation for this great software, it's almost perfect!

One problem I face with Xournal++ that I don't know if it already has a setting for it or it wasn't added yet, is the ability to add a white border/area around a PDF page, sometimes the PDFs I edit are dark color or contain a lot of images that leaves me with no space to add notes, is there a such option as zooming out and getting a white border around the PDF page?

**Comment by bhennion** [comment]
This is not possible at this point. This has been requested already (see #3430 and #1698) and it would certainly be a great addition.
